docs/catalog/cloud/compress-stored-data.md

```diff
@@ -0,0 +1,37 @@
+---
+version: 1.0
+submitted_by: franziska-warncke
+published_date: TBD
+category: cloud
+tags: 
+ - cloud
+ - size:medium
+---
+
+# Compress stored data
+
+## Description
+
+Storing much uncompressed data can result in unnecessary bandwidth waste and increase the storage capacity requirements.
+
+## Solution
+
+Use the right compression tool for the use case to reduce the storage requirements, including both capacity and required bandwidth to write or retrieve data. 
+
+## SCI Impact
+
+`SCI = (E * I) + M per R`  
+[Software Carbon Intensity Spec](https://grnsft.org/sci)
+
+Concerning the SCI equation, using compression when storing data impact one part:
+
+- `E`: Decrease the amount of storage and therefor less E for the storage. However, we should be aware that there may be a slight increase in energy consumed due to compressing and de-compressing data.
+
+## Assumptions
+
```

Comment:
One assumption could be, that you have the ability to choose. If you store much, much data you are forced to use compression to keep storage costs reasonable.
Another one: If you are storage space constrainted you are force to use a high compression (even when there is high CPU required) and also the other way around.